src/coprocessor/dag/expr/builtin_string.rs

```diff
@@ -3293,14 +3289,22 @@ mod tests {
             ("F", "abcdefg", 0),
             ("cd", "abcdefg", 3),
             (" ", "abcdefg", 0),
-            ("", "", 0),
+            ("", "", 1),
```

Comment:
How about `(" ", "")`?

Reply:
("","") return 1; instr_binary("","")
(" ","") return 0;  instr_binary(""," ")
(""," ") return 1;   instr_binary(" ","")

Reply:
It's better to add these cases into the test.
The rest LGTM.

Reply:
> It's better to add these cases into the test.
> The rest LGTM.

@rleungx Thanks for the advice. Already add.